File: src/main/java/com/anod/appwatcher/ListExportActivity.java
package com.anod.appwatcher;

import android.os.Bundle;
import android.support.v4.app.FragmentActivity;

import com.anod.appwatcher.backup.DbBackupManager;

public class ListExportActivity extends FragmentActivity {
	private DbBackupManager mBackupManager;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		
		setContentView(R.layout.activity_list_import);

		mBackupManager = new DbBackupManager(this);

	}

	public DbBackupManager getBackupManager() {
		return mBackupManager;
	}

}

File: src/main/java/com/anod/appwatcher/ListExportActivity.kt
package com.anod.appwatcher

import android.os.Bundle
import androidx.fragment.app.FragmentActivity

import com.anod.appwatcher.backup.DbBackupManager
import com.anod.appwatcher.utils.Theme
import info.anodsplace.framework.app.CustomThemeActivity
import info.anodsplace.framework.app.CustomThemeColors

class ListExportActivity : FragmentActivity(), CustomThemeActivity {

    override val themeRes: Int
        get() = Theme(this).themeDialog
    override val themeColors: CustomThemeColors
        get() = Theme(this).colors

    lateinit var backupManager: DbBackupManager
        private set

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list_import)

        backupManager = DbBackupManager(this)
    }
}
